src/NodaTime.Web/Markdown/root/versions.md

```diff
@@ -4,6 +4,21 @@ User-visible changes from 1.0.0-beta1 onwards. See the
 [project repository](https://github.com/nodatime/nodatime) for more
 details.
 
+## 2.2.1, released 2017-10-14 with tzdb2017b
+
+Bug-fix release:
+
+- Fixes [issue 975]: `LocalInstant.SafeMinus` gives incorrect result around the start/end of time
```

Comment:
Since `LocalInstant` isn't public, is there a way to describe what the end-user of this was, if any? (I was just having a look myself, and it looks like perhaps it was mostly theoretical unless you had crazy zone transitions?)

Reply:
Also it should be 957 :)

Will make it more relevant to end users, but still vague: "Time zone calculations could give incorrect results near the start/end of time"